README.md

```diff
@@ -108,25 +108,12 @@ import (
 ...
 ```
 
-### Pinning to the `v1` releases
+### Using `v1` releases
 
-Similarly to the section above describing use of the `v2` branch, if one wants
-to avoid any unexpected compatibility pains once `v2` becomes `master`, then
-pinning to `v1` is an acceptable option, e.g.:
+Use the github URL to use the v1 branch.  This is illustrated in the Getting Started section below.
 
-```
-$ go get gopkg.in/urfave/cli.v1
-```
-
-``` go
-...
-import (
-  "gopkg.in/urfave/cli.v1" // imports as package "cli"
-)
-...
-```
+Do not use the gopkg.in URL as it does not work for versions >= 1.21.0.
```

Comment:
```suggestion
```

I would remove this line entirely. It might be enough to say _"do this correct thing"_ (eg. use the github url) and avoid saying _"don't do this wrong thing"_ (eg. don't use the gopkg.in url). If it turns out people are still mentioning gopkg.in a bunch, then we can remove it.